Quadriga/src/main/java/edu/asu/spring/quadriga/service/textfile/mapper/impl/TextFileShallowMapper.java

```diff
@@ -0,0 +1,50 @@
+package edu.asu.spring.quadriga.service.textfile.mapper.impl;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import edu.asu.spring.quadriga.dao.textfile.ITextFileDAO;
+import edu.asu.spring.quadriga.domain.workspace.ITextFile;
+import edu.asu.spring.quadriga.dto.TextFileDTO;
+import edu.asu.spring.quadriga.service.textfile.mapper.ITextFileShallowMapper;
+
+@Service
+public class TextFileShallowMapper implements ITextFileShallowMapper {
+
+    
+    @Autowired
+    private ITextFileDAO tfDAO;
+    
+    @Override
+    public List<ITextFile> getTextFileListbyWsId(String wsId) {
+        // TODO Auto-generated method stub
+        return null;
+    }
+
+    @Override
+    public ITextFile getTextFile(String textId) {
+        // TODO Auto-generated method stub
+        return null;
```

Comment:
you can implement that already, even if you don't use it yet. also, I would give it the complete textfileDTO. This class does most likely not need to know the DAO class.